country-specific formatting of numbers

spitzlbergerj opened the pull request:
Hi, Sebastian,

thanks for the note on formatting. I could have thought of it myself ;-)

Many greetings

Sepp

spitzlbergerj committed:
country-specific formatting of numbers

spitzlbergerj committed:
country-specific formatting of numbers

spitzlbergerj committed:
link to Magic Mirror Module MMM-Homematic-Heaters

Sickboy78 commented:
Hi Josef,

I merged it and I changed the default locale to `config.language`.
That way you'll get your global language setting as the default locale, but you can still change it, if you want.

Thx for your contribution & Greetings
Sebastian

spitzlbergerj commented:
Thanks Sebastian, I'll test it out.  Now it's time for a holiday first :-)

________________________________
Von: Sebastian <notifications@github.com>
Gesendet: Samstag, 23. Februar 2019 22:56
An: Sickboy78/MMM-Homematic
Cc: josef@spitzlberger.de; Author
Betreff: Re: [Sickboy78/MMM-Homematic] country-specific formatting of numbers (#4)


Hi Joseph,

I merged it and I changed the default locale to config.language.
That way you'll get your global language setting as the default locale, but you can still change it, if you want.

Thx for your contribution & Greetings
Sebastian

-
You are receiving this because you authored the thread.
Reply to this email directly, view it on GitHub<https://github.com/Sickboy78/MMM-Homematic/pull/4#issuecomment-466701013>, or mute the thread<https://github.com/notifications/unsubscribe-auth/AlLXGovGIhPVdf65urcmw6wLVoiMGpG5ks5vQbkAgaJpZM4azII_>.

Sickboy78 commented:
Have a beautiful holiday!

spitzlbergerj commented:
Hi, Sebastian,

i have now renamed my fork from MMM-Homematic-Heaters to spitzlbergerj/MMM-Homematic-Thermostats, after the original author apparently doesn't maintain the "heaters" anymore. Do you want to change the reference to my fork in your readme? This will certainly have time until the next change.

The icons are very nice. Where did you get the pictures from? I still need an icon for "presence".

Thank you very much.
greetings
Sepp

Von: josef@spitzlberger.de <josef@spitzlberger.de>
Gesendet: Sonntag, 24. Februar 2019 05:32
An: Sickboy78/MMM-Homematic <reply@reply.github.com>
Betreff: Re: [Sickboy78/MMM-Homematic] country-specific formatting of numbers (#4)

Thanks Sebastian, I'll test it out.  Now it's time for a holiday first :-)

________________________________
Von: Sebastian <notifications@github.com<mailto:notifications@github.com>>
Gesendet: Samstag, 23. Februar 2019 22:56
An: Sickboy78/MMM-Homematic
Cc: josef@spitzlberger.de<mailto:josef@spitzlberger.de>; Author
Betreff: Re: [Sickboy78/MMM-Homematic] country-specific formatting of numbers (#4)


Hi Joseph,

I merged it and I changed the default locale to config.language.
That way you'll get your global language setting as the default locale, but you can still change it, if you want.

Thx for your contribution & Greetings
Sebastian

-
You are receiving this because you authored the thread.
Reply to this email directly, view it on GitHub<https://github.com/Sickboy78/MMM-Homematic/pull/4#issuecomment-466701013>, or mute the thread<https://github.com/notifications/unsubscribe-auth/AlLXGovGIhPVdf65urcmw6wLVoiMGpG5ks5vQbkAgaJpZM4azII_>.

Sickboy78 commented:
Hi Sepp,

I made the icons myself. I added one for presence, hope you like it.
I also changed the link to MMM-Homematic-Heaters.

Best Regards
Sebastian

spitzlbergerj commented:
Hi Sebastian,

very nice icon. Thanks, also for the entry to MMM-Homematic-Thermostats

Bye
Sepp

Von: Sebastian <notifications@github.com>
Gesendet: Sonntag, 24. März 2019 15:53
An: Sickboy78/MMM-Homematic <MMM-Homematic@noreply.github.com>
Cc: josef@spitzlberger.de; Author <author@noreply.github.com>
Betreff: Re: [Sickboy78/MMM-Homematic] country-specific formatting of numbers (#4)


Hi Sepp,

I made the icons myself. I added one for presence, hope you like it.
I also changed the link to MMM-Homematic-Heaters.

Best Regards
Sebastian

—
You are receiving this because you authored the thread.
Reply to this email directly, view it on GitHub<https://github.com/Sickboy78/MMM-Homematic/pull/4#issuecomment-475966556>, or mute the thread<https://github.com/notifications/unsubscribe-auth/AlLXGn7pRPTn4vN5CxKdTmpR4Sb-1oDaks5vZ5E-gaJpZM4azII_>.

spitzlbergerj commented:
Hallo Sebastian,

ich plane in den nächsten Wochen noch eine Erweiterung Deines Moduls. Vorab möchte ich dich aber fragen, ob das in Deinem Sinne ist oder ob ich das Ganze in ein eigenes Modul packen soll, das dann aber auf Deiner Homematic-Schnittstelle basieren würde.

Ich möchte mir auf dem Spiegel eine Reihe von Zuständen von IP-Geräten (mein Server, meine unterschiedlichen NAS, ggf. PCs) anzeigen. Die Zustände sind in der Homematic als Variablen hinterlegt. Ich stelle mir da Ampelfarben vor. Es soll also eine Zeile geben, in der nacheinander mehrere "farbig gefüllte kleine Kreise" angezeigt werden. Grüne alles in Ordnung, Gelb Maßnahme notwendig, rot ist nicht erreichbar oder so. Ggf. steht dann drüber, welches Gerät von dem Punkten symbolisiert werden.

Beispiel:

[cid:image001.png@01D4FF4C.08E093C0]


Das ist sicherlich in Deinem Modul möglich, ändert aber dessen Ausrichtung etwas. Bisher war eine Zeile immer ein Zustand. Jetzt wären in einer Zeile unter Umstände eine Reihe von unterschiedlichen Zuständen.

Was ist Dir denn lieber? In dein Modul einbauen oder dein Modul als Basis nehmen und ein eigenes draus machen?

Viele Grüße

Sepp




--------------------------------------------------------------------
Josef J. Spitzlberger               E-Mail:  josef@spitzlberger.de<mailto:josef@spitzlberger.de>
Schloss Lustheim 1                  Telefon: 089 / 24 29 51 06
85764 Oberschleißheim               Telefax: 089 / 24 29 51 07
--------------------------------------------------------------------
